swagger.yaml

```diff
@@ -3105,6 +3105,19 @@ definitions:
               format: uuid
             hostname:
               type: string
+      selected-installation-disk-paths:
+        type: array
+        x-go-custom-tag: gorm:"type:varchar(64)[]"
```

Comment:
I don't understand this ... If the item in the array is an object, why does the array itself have a custom tag?

Reply:
This feels like it would limit the length of the json representation of each object to 64 chars which, while possible, seems error-prone and unnecessary.

Reply:
Then it should be removed from all the host-related items in the UpdateCluster response.

Reply:
this tag is not really doing anything because this definition don't really exists in the DB gorm will not do anything with it and it can be removed.

Reply:
agreed